README.md

```diff
@@ -162,3 +162,23 @@ You can run automated tests, code analysis and Django configuration checks with:
 ```
 
 Always run this command before submitting code in a pull request and make sure that there are no warnings or failed tests.
+
+
+### Running MiddleMan
+
+You can run MiddleMan service with default values with:
```

Comment:
I'd add a sentence or two about what the Middleman is. My proposal:

```` markdown
### Running Middleman
Concent signs Ethereum transactions by passing them to an external signing service provided by Golem. 
To decrease the attack surface the service is not serving any requests.
Instead Concent runs a component that opens a TCP port an allows the signing service to connect at will.
This component (internally called "Middleman") needs to be started separately from the main server:

``` bash
concent_api/manage.py middleman
```

The command should work fine without any extra arguments.
You can use `--help` option to see all the available options.

Note that in development you need to run an instance of the Signing Service yourself.
The application is maintained by the Concent team as well and you can find it in this repository.
See [Signing Service README](signing_service/README.md)
````

BTW, please make sure the relative link at the end works as expected. According to [Relative links in markup files](https://blog.github.com/2013-01-31-relative-links-in-markup-files/) it should work if you put it in the README.

Reply:
I like your proposal :) I have added some colons and fixed typos. Done :+1:

Reply:
One more tiny thing: let's call it `Middleman` (not `MiddleMan`) for consistency. In your paragraph you're using both :)